docs/deploy-event-router.md

```diff
@@ -0,0 +1,106 @@
+# Deploying Event Router
+The EventRouter is a simple service that listens to kubernetes evenets and writes them to `STDOUT`.  It is not managed by the `cluster-logging-operator`.
+Creating a deployment using the following template will enable event collection equivalent to the same capability from 3.11 releases or earlier.
+Currently, all events are collected by the cluster logging collector and indexed to the `infra` index when
```

Comment:
Please review the fluent.conf to see if this is a correct statement

Reply:
```
<rule>
  key CONTAINER_NAME
  pattern ^k8s_[^_]+_logging-eventrouter-[^_]+_
  tag kubernetes.journal.container._default_.kubernetes-event
</rule>

<match **_default_** **_kube-*_** **_openshift-*_** **_openshift_** journal.** system.var.log**>
  @type relabel
  @label @_LOGS_INFRA
</match>
```

Observed k8s events in Kibana's infra-000001 index pattern.